wdl-lint/tests/lints/preamble-comment-after-version/source.errors

```diff
@@ -1,15 +1,25 @@
+note: unnecessary whitespace in document preamble [rule: PreambleWhitespace]
+  ┌─ tests/lints/preamble-comment-after-version/source.wdl:1:38
+  │  
+1 │   ## This is a normal preamble comment.
+  │ ╭─────────────────────────────────────^
+2 │ │        ## Bad leading whitespace!
+  │ ╰───────^
```

Comment:
How hard would it be to drop the leading newline from the reported span? It's being indicated as problematic, but the newline is fine (and is actually needed). The span should only cover the literal spaces before the comment.

Reply:
I should be able to remove it.